docs/search_engine_optimization/seo.md

```diff
@@ -0,0 +1,38 @@
+---
+description: SEO can help your content score higher in search results, which leads to greater visibility.
+---
+
+# Understand SEO
+
+Every time users search online, they are presented with a list of results, 
+together with page title and a short summary. 
+Based on this limited information they choose which link to follow. 
+The more your page stands out, the more likely users are to pay you a visit.
+
+With Search Engine Optimization (SEO) tool, you can optimize your website for both 
+visitors and search engines. 
+SEO brings in more organic traffic and improves the website's visibility in 
+Search Engines Results Pages (SERPs). 
+It is a core feature of Digital Experience Platform. 
+
+With SEO you can also control the appearance of search results on SERPs, by 
+providing the intended wording for the title, description, keywords, etc.
+
+## Meta Tags
+
+`Title`, `Description` and `Keywords` tags are displayed on SERPs.
+They also help the search engine figure out what the content is about and how 
+high it should rank. 
+A `Canonical` tag is used by the search engine's internal algorithm. 
+Once you define the tags for your content, they are rendered as part of the `<HEAD>` 
+section of a web page, from where the search engine intercepts their values.
+
+- Title is displayed as a prominent heading in the search. You want to make it relevant and attractive for users to click on it. 
+- Description appears next to or below the title. Here you tell visitors what the content is about. Your goal is to optimize it for better visibility both on desktop and mobile.
+- Keywords can help the search engine position the results.
+- Canonical URL tells search engines tell search engines that the master copy of content resides at a specific URL, to prevent duplicates from competing against each other in search results.
```

Comment:
```suggestion
- Canonical URL tells search engines that the master copy of content resides at a specific URL, to prevent duplicates from competing against each other in search results.
```